src/runner.py

```diff
@@ -95,6 +94,10 @@ def _schedule_test(self, tarball_node, plan, device):
 class RunnerLoop(Runner):
     """Runner subclass to execute in a loop"""
 
+    def __init__(self, configs, args):
+        self._job_tmp_dirs = {}
+        super().__init__(configs, args)
```

Comment:
Ideally `super()` should be called first, to follow object-oriented programming best practices.  We could also have `*args, **kwargs` in the constructor signature to pass that to the parent constructor without having to redefine individual arguments since we're not using them here.